### ISSUE TITLE
GRSciColl question/idea - field for link to contact form

### ISSUE THREAD

**Issue by ManonGros** [gbif/registry#408]
I am logging the issue to not forget it.
One of the institutions on GRSciColl would like to encourage users to contact them via their online form ( https://www.qm.qld.gov.au/About+Us/Contact+Us) instead of by email.
However, there isn't really a place for this type of link.

I wonder if we should consider adding such field in the future. Or maybe check with the TDWG CDs if this is something that their model can accommodate for.

**Comment by ManonGros** [comment]
Yes they could use another free text field. The problem is that some have used links instead of email addresses in IH so we aren't getting updates for these.
I imagine it would be messy to allow URLs or email addresses for the same field. I will close the issue and revisit if we get more requests like that.

**Comment by marcos-lg** [comment]
ok, if it happens often it can also be handled in the IH sync to convert those links in the email field into comments or sth like that.
